Code/Tools/ProjectManager/Source/ProjectUtils.cpp

```diff
@@ -59,29 +60,63 @@ namespace O3DE::ProjectManager
             return false;
         }
 
+        static bool SkipFilePaths(const QString& curPath, QStringList& skippedPaths, QStringList& deeperSkippedPaths)
```

Comment:
The `AZ::IO::PathView::IsRelativeTo` function can be used to determine if each `skippedPaths` is a prefix of the `curPath`

Reply:
Alternatively if you want to only use Qt logic for excluding a folder, you can look at the logic in [assetScannerWorker.cpp](https://github.com/o3de/o3de/blob/development/Code/Tools/AssetProcessor/native/AssetManager/assetScannerWorker.cpp#L103-L111) where the project cache directory is skipped.
That can adapted to this general function

Reply:
Thanks for the suggestions but I'd prefer to keep the more generic solution I made here for now at least. It has the unit tests to support that it works. Also was interested in the second suggestion but it doesn't seem completely straight forward to pull out of the asset processor. Also maybe we'll want to skip deeper directories later and or specific files.